dracut/modules.d/90kiwi-live/module-setup.sh

```diff
@@ -27,7 +27,7 @@ install() {
     local dmsquashdir=
     inst_multiple -o checkmedia
     inst_multiple \
-        umount dmsetup blockdev blkid lsblk dd losetup \
+        umount dmsetup partx blkid lsblk dd losetup \
```

Comment:
Isn't a similar change also required to `dracut/modules.d/99kiwi-lib/module-setup.sh`. I think `partx` should be added to list for `kiwi-lib` module too.

Reply:
@davidcassany It's so good that you are also looking at this. Yes I missed to add it to the kiwi-lib module. Thanks a ton. I think we are good now. Can you do a final review ? Thanks much